packages/scenes/src/components/layout/grid/types.ts

```diff
@@ -12,4 +12,9 @@ export interface SceneGridItemStateLike extends SceneGridItemPlacement, SceneObj
   isDraggable?: boolean;
 }
 
-export type SceneGridItemLike = SceneObject<SceneGridItemStateLike>;
+export interface SceneGridItemLike extends SceneObject<SceneGridItemStateLike> {
+  /**
+   * This is needed by PanelRepeaterGridItem to have special css behavior for responsive (mobile) breakpoint.
```

Comment:
Just a nit, let's be core Grafana agnostic when describing why something is available in the public API.
```suggestion
   * Provide a custom CSS class name to the underlying DOM element when special styling (i.e. for mobile breakpoint) is required
   * ```